hugegraph-dist/scripts/validate-release.sh

```diff
@@ -25,6 +25,7 @@ URL_PREFIX="https://dist.apache.org/repos/dist/dev/incubator/hugegraph/"
 # release version (input by committer)
 RELEASE_VERSION=$1
 JAVA_VERSION=$2
+USER=$3
```

Comment:
also move this file to hugegraph-doc repo?

Reply:
> also move this file to hugegraph-doc repo?

It can be discussed that the server keep validate shell script while hugegraph-doc keep ci actions ?
@javeme @imbajin

Reply:
> > also move this file to hugegraph-doc repo?
> 
> It can be discussed that the server keep validate shell script while hugegraph-doc keep ci actions ? @javeme @imbajin

Both fine, and wait until the **first release** is completed. After confirming the stability, we can merge current PR, 
could leave it alone for now